adage/adage/config.py.template

```diff
@@ -131,7 +131,9 @@ AWS_CONFIG = {
         '<configure:http://example.com/tribe_client/get_token>',
     'tribe_scope': 'write',
     'tribe_login_redirect': '/#/home',
-    'tribe_logout_redirect': '/#/home'
+    'tribe_logout_redirect': '/#/home',
+    'public_geneset_folder': os.path.join(OS_CONFIG['home_dir'],
```

Comment:
Ah yes - so this is actually to be able to access the genesets that are pickled from Tribe for this geneset enrichment analysis, and it is called by the tribe-client package. I totally agree this is unclear, so I will add some comments stating that purpose.